Assets/Scripts/Combat/PossessionManager.cs

```diff
@@ -187,6 +199,11 @@ IEnumerator SpawnPlayer()
         yield return null;
     }
 
+    void SpawnPlayerPossessObject(PlayerMovementController player)
+    {
+        Instantiate(slimePossessPrefab, player.transform.position, player.transform.rotation).GetComponent<SlimePossessMotion>().SetTarget(slimePossessionSpriteRenderer, slimeAttachPoint.position, playerMovementController.IsFacingRight());
```

Comment:
Minor note (for the future): instead of Instantiating a prefab, you can also Instantiate a component and avoid having to immediately call GetComponent on what was spawned:

```
[SerializeField] SlimePossessMotion possessMotionPrefab;
// ...
Instantiate(slimePossessPrefab, player.transform.position, player.transform.rotation).SetTarget(slimePossessionSpriteRenderer, slimeAttachPoint.position, playerMovementController.IsFacingRight());
```

Haven't tested this, but I think this works. If you're curious, see the [last code example in the docs](https://docs.unity3d.com/ScriptReference/Object.Instantiate.html).

Reply:
Yep, it works, thanks